cmd/peer-s3-client.go

```diff
@@ -185,6 +186,21 @@ func (sys *S3PeerSys) ListBuckets(ctx context.Context, opts BucketOptions) ([]Bu
 				}
 			}
 		}
+		// loop through buckets and see if some with lost quorum
+		// these could be stale buckets lying around, remove them
+		for bktName, count := range bucketsMap {
+			if count < quorum {
+				// Queue a bucket heal task
+				globalHealStateLK.Lock()
+				bgSeq, ok := globalBackgroundHealState.getHealSequenceByToken(bgHealingUUID)
```

Comment:
We should put this to MRF not global heal queue @shtripat 

A much simpler code to handle it's called er.addPartial()

Reply:
Done